index.html

```diff
@@ -0,0 +1,557 @@
+                // console.log('prev from product');
+                // console.log(carouselWidth)
+                // console.log(cardWidth)
+                // console.log(scrollPosition)
+                if (scrollPosition > 0) {
+                    scrollPosition -= cardWidth;
+                    $("#prodcar").animate({
+                            scrollLeft: scrollPosition
+                        },
+                        500
+                    );
+                }
+            });
+        } else {
+            $(multipleCardCarousel).addClass("slide");
+        }
+
+        var multipleCardCarousel = document.querySelector("#beauty");
+        var scrollPosition1 = 0;
+        if (window.matchMedia("(min-width: 768px)").matches) {
+            var carousel = new bootstrap.Carousel(multipleCardCarousel, {
+                interval: false,
+            });
+            var carouselWidth = $("#beautycar")[0].scrollWidth;
+            var cardWidth = $("#beauty .carousel-item").width();
+            $("#beauty .carousel-control-next").on("click", function () {
+                if (scrollPosition1 < carouselWidth - (cardWidth * 4)) {
+                    scrollPosition1 += cardWidth;
+                    $("#beautycar").animate({
+                            scrollLeft: scrollPosition1
+                        },
+                        500
+                    );
+                }
+            });
+            $("#beauty .carousel-control-prev").on("click", function () {
+                if (scrollPosition1 > 0) {
+                    scrollPosition1 -= cardWidth;
+                    $("#beautycar").animate({
+                            scrollLeft: scrollPosition1
+                        },
+                        500
+                    );
+                }
+            });
+        } else {
+            $(multipleCardCarousel).addClass("slide");
+        }
+
+        // Set the date we're counting down to
+        var countDownDate = new Date("March 23, 2023 11:16:00").getTime();
+
+        // Update the count down every 1 second
+        var x = setInterval(function () {
+
+            // Get today's date and time
+            var now = new Date().getTime();
+
+            // Find the distance between now and the count down date
+            var distance = countDownDate - now;
+            console.log(distance);
+           
+            // Time calculations for days, hours, minutes and seconds
+            var days = Math.floor(distance / (1000 * 60 * 60 * 24));
+            var hours = Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
+            var minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60));
+            var seconds = Math.floor((distance % (1000 * 60)) / 1000);
+
+            // Output the result in an element with id="demo"
+            document.getElementById("timer").innerHTML = days + "d " + hours + "h " +
```

Comment:
Why you have written javascript code? You have already used jquery then why you have used javascript here. Provide some explanation here.

Reply:
Actually I have taken a reference for the timer function from external sources. I found implementation with javascript first and I added that in my code. No other specific reasons. But if you say so, I'll change it to the JQuery.